tests/integration/service_test.py

```diff
@@ -902,7 +902,7 @@ def test_log_drive_empty_default_jsonfile(self):
         self.assertFalse(log_config['Config'])
 
     def test_log_drive_none(self):
-        service = self.create_service('web', log_driver='none')
+        service = self.create_service('web', logging={'driver': 'none'})
```

Comment:
I think we want to keep these existing tests and just rename them to be V1 tests, and add new ones for the V2 format